docs/insomnia/requests.md

```diff
@@ -0,0 +1,97 @@
+---
+layout: article-detail
+title:  Requests
+category: "Requests and Responses"
+category-url: requests-and-responses
+---
+
+Insomnia supports sending requests via [HTTP method](https://developer.mozilla.org/en-US/docs/Web/HTTP/Methods), [gRPC](https://grpc.io/docs), and [GraphQL](https://graphql.org).
+
+To learn how to repurpose parts of your request, see
+[Environment Variables](/insomnia/environment-variables).
+
+## Send an HTTP method request
+
+Send an HTTP method request in a Document or Collection. The following instructions
+assume you're starting with an empty Document or Collection.
+
+In addition to standard HTTP methods, Insomnia supports [Custom HTTP Methods](#custom-http-methods).
+
+1. In a Document, select the **Debug** tab, then **New Request**. In a Collection, select **New Request**.
+1. In the **New Request** modal:
+   1. Enter a request name.
+   1. Select a method from the dropdown.
+   1. Click **Create**.
+1. Enter the API URL in the request URL input.
+1. To add authentication, go to the **Auth** tab. Use the **Auth** dropdown to select your authentication method.
+1. When you have entered your request URL and authentication, click **Send**.
+
+See [Request Options](#request-options) to customize your request.
+
+## Send a gRPC request
```

Comment:
I suggest having the order of sections be:

* Send an HTTP request
	* Custom HTTP methods
	* Request Options
* Send a GraphQL request
* Send a gRPC request
	* gRPC request options

This is because GraphQL is very similar to HTTP, especially in the Insomnia UI but also in practice GraphQL is built on top of the same protocol. gRPC however is entirely separate.